src/Host/Client/Impl/Definitions/PlotMessage.cs

```diff
@@ -17,11 +17,11 @@ public struct PlotMessage {
 
         public bool IsClearAll => string.IsNullOrEmpty(FilePath);
 
-        public bool IsPlot => new FileInfo(FilePath).Length > 0;
+        public bool IsPlot => File.Exists(FilePath) && new FileInfo(FilePath).Length > 0;
```

Comment:
Technically, file can exists during first check and be deleted during second change. We need to wrap `new FileInfo(FilePath).Length > 0` into try/catch